src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -206,6 +207,12 @@ public function getEnv(string $prefix, string $name, \Closure $getEnv): mixed
             return null;
         }
 
+        if ('shuffle' === $prefix) {
+            shuffle($env);
```

Comment:
we should validate that $env is an array before doing the call, and throw a RuntimeException if that's not the case